### ISSUE TITLE
Facebook App Review

### ISSUE THREAD

**Issue by mayfeld** [postyou/contao-facebook-connector_basic#7]
What to do with the necessary Review of apps in the developer, from August, the Api should otherwise be blocked. I am supposed to use user_link, user_gender, user_age_range, 
specify an configure
![bildschirmfoto 2018-05-08 um 17 27 32](https://user-images.githubusercontent.com/5961313/39766667-1f15ecee-52e5-11e8-93ec-06250a38a76e.png)

**Comment by theDyingMountain** [comment]
The review process for a contao facebook sample app, is started some days ago. Now I'm waiting for an answer by facebook.

**Comment by Metis77** [comment]
I guess access to "Page Public Content Access" needs to be requested in addition?

**Comment by theDyingMountain** [comment]
Yes "Page Public Content Access" is included in ongoing review process.

**Comment by Metis77** [comment]
Any news on this?

**Comment by theDyingMountain** [comment]
No sorry, no response from facebook since 9th may...
